README.md

```diff
@@ -37,12 +37,17 @@ See the official [React Native website](https://facebook.github.io/react-native/
 ### Requirements
 
 In addition to the requirements from the [Getting Started guide](http://facebook.github.io/react-native/docs/getting-started.html):
+- [Windows 10 Professional](https://www.microsoftstore.com/store/msusa/en_US/pdp/Windows-10-Pro/productID.319935900)
+  - Windows 10 Home cannot run the Windows Phone Emulator, but you would be able to test on a Windows Phone device.
```

Comment:
I don't think Pro is required to run UWP desktop apps, just the Windows Phone emulator.